email_alert_service/presenters/unpublishing_message_presenter.rb

```diff
@@ -51,4 +53,14 @@ def url_for(base_path:, **params)
     uri.query = URI.encode_www_form(query).presence
     uri.to_s
   end
+
+  def page_summary
+    description = subscriber_list.fetch("description")
+    if description.nil? || description.empty?
```

Comment:
I think `description = subscriber_list["description"]` would read slightly better here, given that we're then checking for validity in the condition.  I think it can often be assumed that, if `foo = bar.fetch("baz")`, then `foo` is not `nil` (which is incorrect, but easy to arrive at).

Reply:
Sure, I can change that